backend/wmg/data/cube_pipeline.py

```diff
@@ -75,13 +81,19 @@ def load_data_and_create_cube(
     logger.info("Loaded datasets into corpus")
     create_cubes(corpus_path)
     logger.info("Built expression summary cube")
-
+    if validate_cubes:
+        is_valid = Validation(corpus_path).validate_cube()
+        if is_valid is False:
+            return
```

Comment:
should it return boolean? The pipeline `main` block could return as `sys.exit(1)` if validation fails. Not sure if that's helpful in the grand scheme of things, but maybe it marks the batch job as failed? _Is_ failed validation to be considered a job failure?

Reply:
Retracting this comment, since validation is only to be used for ad hoc runs